### ISSUE TITLE
Displaying media in the summary (hierarchy view)

### ISSUE THREAD

**Issue by alyblenkin** [getodk/collect#6084]
Blocked on #6234 
Blocked on #6470 

Our aim is to improve the user experience for both the inline and hierarchy views and allow them to share code. 

## User Stories

As a data collector, 
I want to view the images I’ve captured in the summary view 
so I can ensure everything is correct before submitting my form.

As a data collector, 
I want to access the images in my sent forms 
so I can verify what I’ve submitted.

## **Acceptance Criteria**

**Summary View:**
Given I am on the summary view,
When I see the thumbnail,
Then I can tap to expand and verify the image is correct.

**Sent Forms:**
Given I am viewing a sent form,
When I review the submission,
Then I see a read-only view of my media.

**Form Entry:**
Given I am filling out a form,
When I take a photo,
Then the image is displayed in a way that doesn't take up the entire screen(fixing the height to 300px and width flexible to natural size of the image).

## Priority order & scope

These don't have to be done as part of one change: each media type could be added in isolation.

- **images** - as described above 
- **geo** - improve how we show the coordinates (left align because it's easier to read, and take out "long" and "lat" labels)
- **audio** - simple progress bar style for audio 
- **barcodes** - displaying an icon with the number 
- **files** - if we can show a preview of a file (alternatively we could just show an icon)
- **video** - We could do the same experience for file and video with an icon, launch whatever player Android has 

### Out of scope 
- **geo** - displaying a map with lines, polygons or points, 
- **audio soundwave** - something we may get to later if people find it useful 

[Figma ](https://www.figma.com/file/KFi9hIQdrRgo0rLqIC7Sg8/ODK-user-flows?type=design&node-id=4799%3A93007&mode=design&t=DPp0BGAhIfEUDyKY-1)
